packages/accounts-base/accounts_server.js

```diff
@@ -176,9 +176,10 @@ Ap._failedLogin = function (connection, attempt) {
   });
 };
 
-Ap._successfulLogout = function () {
+Ap._successfulLogout = function (connection, userId) {
+  const user = userId && this.users.findOne(userId);
```

Comment:
Are we sure we want to always make a database query here? Why wouldn't we just pass `userId` and leave to the developer to fetch the user document if they want? I think this is not necessary to do automatically.